physics/progsigma_calc.f90

```diff
@@ -34,8 +34,8 @@ subroutine progsigma_calc (im,km,flag_init,flag_restart,           &
 !     intent out
       real(kind=kind_phys), intent(out) :: sigmaout(im,km)
       real(kind=kind_phys), intent(out) :: sigmab(im)
-      character(len=*),     intent(out) :: errmsg
-      integer,              intent(out) :: errflg
+      character(len=*),     intent(inout) :: errmsg
```

Comment:
errmsg and errflg are both defined but never set in this file, so 'INOUT' allows that to pass through this subroutine.

Reply:
The CCPP rules are that any scheme must set these variables. However, since `[physics/progsigma_calc.f90](https://github.com/ufs-community/ccpp-physics/pull/170/files/77f3a4f1c0cdbb4fe61cd5640e64a0b6bf07e529#diff-9e2ad91c9f4e1563c831cf9a10a2634614e3d931e2f1ee2a15bcce39c7fd6089)` is not a CCPP entry point but a subroutine getting called within a CCPP scheme, your solution is correct I think.